core/vision/src/main/java/com/arcrobotics/ftclib/vision/UGBasicHighGoalPipeline.java

```diff
@@ -73,43 +89,36 @@ public Mat processFrame(Mat input) {
 
         Imgproc.findContours(blueThreshold, blueContours, new Mat(), Imgproc.RETR_TREE, Imgproc.CHAIN_APPROX_SIMPLE);
         Imgproc.findContours(redThreshold, redContours, new Mat(), Imgproc.RETR_TREE, Imgproc.CHAIN_APPROX_SIMPLE);
+        blueContours = blueContours.stream().filter(i -> filterContours(i) && (((double) Imgproc.boundingRect(i).width / Imgproc.boundingRect(i).height) > 1) &&(((double) Imgproc.boundingRect(i).width / Imgproc.boundingRect(i).height) < 2)).collect(Collectors.toList());
+        redContours = redContours.stream().filter(i -> filterContours(i) && (((double) Imgproc.boundingRect(i).width / Imgproc.boundingRect(i).height) > 1) &&(((double) Imgproc.boundingRect(i).width / Imgproc.boundingRect(i).height) < 2)).collect(Collectors.toList());
```

Comment:
These are long lines. Can they not be broken down and/or simplified or split into multi-line?

Reply:
I'm not sure :(

Reply:
Actually, I can give it a body.